src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetCultureCommand.cs

```diff
@@ -1,23 +1,71 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Globalization;
 using System.Management.Automation;
 
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
     /// Returns the thread's current culture.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "Culture", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
+    [Cmdlet(VerbsCommon.Get, "Culture", DefaultParameterSetName = NameParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=113312")]
     [OutputType(typeof(System.Globalization.CultureInfo))]
     public sealed class GetCultureCommand : PSCmdlet
     {
+        private const string NameParameterSet = "Name";
+        private const string ListAvailableParameterSet = "ListAvailable";
+
+        /// <summary>
+        /// Gets or sets culture names for which CultureInfo values are returned.
+        /// </summary>
+        [Parameter(ParameterSetName = NameParameterSet, Position = 0, ValueFromPipeline = true)]
+        [ValidateNotNull]
```

Comment:
Shouldn't this be ValidateNotNullOrEmpty?

Reply:
For empty string name `InvariantCulture` is returned.

Reply:
@iSazonov - I think that behavior's too subtle.

Additionally, the ProcessRecord doesn't match the class comment (Return's the thread's current culture).  In short, reviewing the code appears to not support getting the current culture.

NOTE: Line 40 that compares Name to null 'looks' like dead code since name must be non-null for the Name parameter set.  Having an explicit or fall through 'default' case statement will make this clearer.

I suggest the following:

1: Retain the original behavior explicitly
Set the DefaultParameterSetName to 'default' and return the thread's current culture.

2: Name parameter set
Either clearly document that an empty string returns InvariantCulture or don't allow empty strings and require explicit names.  

3: ListAvailable
As implemented.

Reply:
I too think that `""` is obscure and suggest requiring explicit use of `Invariant-Culture`, i.e.:  
 
`Get-Culture Invariant-Culture`

Reply:
1 - Fixed.
2- Let's consider the scenario - script asks user to input a culture, user press Enter to choose "Default". Without user typing the script get empty string as "default". I guess CoreFX consider InvariantCulture as default if culture is unknown or not selected. We use InvariantCulture widely in PowerShell code too. So allowing empty string in Get-Culture looks good.

Reply:
Re 2: Curiously, the empty string is not just a placeholder that signals the intent to retrieve the invariant culture when you call `.GetCultureInfo()`, it is the actual value of the `.Name` property  of the culture-info object returned - so I guess it makes sense to surface that.(Unlike what I thought earlier, `"Invariant-Culture"` doesn't actually work; on Unix it implicitly creates an ad-hoc culture, on Windows it breaks.)

Reply:
Perhaps add a comment about this in the code

Reply:
Added.